packages/babel-plugin-proposal-pipeline-operator/test/fixtures/hack-caret/while-statement-with-pipe-in-condition/input.js

```diff
@@ -1,7 +1,7 @@
 let i = 0;
 let sum = 0;
 
-while (i |> (i = ^ + 1) |> ^ <= 10)
+while (i |> (i = 2 * ^ - ^ + 1) |> ^ <= 10)
```

Comment:
This is just to keep the "spirit" of the test, otherwise it was completely inlined.